index.js

```diff
@@ -18,12 +22,29 @@ function onClosed() {
 	mainWindow = null;
 }
 
+function overrideRefreshWindowShotcuts(){
+	/*const ret = */globalShortcut.register('CmdOrCtrl+R', () => {
+		//console.log('CmdOrCtrl+R')
+	})
+	/*const ret1 = */globalShortcut.register('F5', () =>  {
+		//console.log('F5')
+	});
+
+	/*if (!ret || !ret1) {
+		console.log('registration failed')
```

Comment:
I think throwing an error (or at least logging one) here would be beneficial as it is unexpected behaviour?

Reply:
I will change the function and log it if an expected behavior occurs, however it is guaranteed that it will register the shortcut once the app is open per operating system .